packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -457,7 +457,7 @@ class RangeMaintainingScrollPhysics extends ScrollPhysics {
 /// ```dart
 /// BouncingScrollPhysics(parent: AlwaysScrollableScrollPhysics())
 /// ```
-/// (@end-tool}
+/// {@end-tool}
```

Comment:
Given this sort of error, do we want to just search for all words beginning with "@" outside of code blocks? Or is that too many false positives?

Reply:
Unfortunately, that has too many false positives. If that becomes a common source of error, we could also match on something like `(@foo}` and report an error for those.

Reply:
Yeah, If figured. OK.